### PR TITLE
☕  Replace the inventory subgraph with a graphql-java one

### PR DESCRIPTION
**Pull request by martinbonnin**
This is meant as a showcase of https://github.com/apollographql/federation-jvm. It should be functionality equivalent to the previous Apollo Server version except the implementation language is Kotlin and it's using graphql-java and Spring Boot.

One thing to note is that the first run of the container is wayyyyy slower than Apollo Server because it runs Gradle inside the container and this has to download/run a bunch of things. A more real-life example would be to build the jar outside Docker but that'd require more steps.

Ping @BoD

### CODE REVIEW

File: supergraph.graphql [hunk 1]
```diff
@@ -1,134 +0,0 @@
-
```
Comment (prasek) [inline supergraph.graphql:1]: @martinbonnin this is autogenerated but useful when just browsing the demo repo, suggest we leave this in.

**Review by prasek — COMMENTED** [review]
Thanks @martinbonnin looks great. Dropped one comment, but thanks for adding this!

File: supergraph.graphql (hunk 1 above)
Comment (martinbonnin) [reply supergraph.graphql:1]: Oooopsie... very sorry, I didn't mean to commit this. Will rollback. Thanks for the heads up!

**Comment by BoD** [comment]
✅ 👍

**Pull request merged**